server/web-server-stats.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <cstdint>
+
+#include "common/mixin/not_copyable.h"
+#include "common/smart_ptrs/singleton.h"
+#include "common/stats/provider.h"
+
+#include "server/server-stats.h"
+
+struct WebServerStats : vk::not_copyable {
+  using Stats = std::tuple<int64_t, int64_t, int64_t, int64_t>;
+
+  void init() noexcept;
+
+  bool store(ServerStats::WorkersStat const & stats);
+
+  WebServerStats::Stats load() const;
+
+private:
```

Comment:
As it's singleton it should have private default constructor, and friend `singleton<WebServerStats>`

Reply:
fix